### DISCUSSION TITLE
Does ReadySet supports mysql client connect?

### DISCUSSION THREAD

**Discussion by wubx** [readysettech/readyset#37] [Q&A] [answered]
use  MySQL client:
```
ERROR 2013 (HY000): Lost connection to server at 'handshake: reading initial communication packet', system error: 11
```
backend use MySQL 8.0

**Comment by jbensal** [comment]
Hi @wubx.

We do support MySQL 8.0. We'll need a bit more information to help you debug this. Can you share the command you're using to run ReadySet and the logs that you're seeing on that instance?

**Reply by wubx** [reply]
Deploy MySQL use : https://github.com/readysettech/readyset#quickstart  
```
 mysql -h 127.0.0.1 -P3307 -u root -preadyset  testdb
ERROR 2013 (HY000): Lost connection to server at 'handshake: reading initial communication packet', system error: 11
```

**Comment by jbensal** [comment, marked as answer]
Hi @wubx, 

The example docker command we give spins up ReadySet on port 5433 (default for Postgres), but we expose port 3307 from the container and try and connect on that port. Here's a correct command you can use to run ReadySet. Thanks for reporting this, I'll update the instructions to be correct.

```
docker run -d \
--name=readyset \
--publish=3307:3307 \
--platform=linux/amd64 \
--volume='readyset:/state' \
--pull=always \
-e DEPLOYMENT_ENV=quickstart_github \
public.ecr.aws/readyset/readyset:beta-2022-12-15 \
--standalone \
--deployment='github-mysql' \
--database-type=mysql \
--upstream-db-url=mysql://root:readyset@172.17.0.1:3306/testdb \
--address=0.0.0.0:3307 \
--username='root' \
--password='readyset' \
--query-caching='explicit' \
--db-dir='/state'

```

and then connect with 

```
 mysql -h 127.0.0.1 -P3307 -u root -preadyset  testdb
```
